src/Symfony/Component/Finder/Iterator/ReverseSortingIterator.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Finder\Iterator;
+
+/**
+ * Reverse the order of a previous iterator.
+ *
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+class ReverseSortingIterator extends \ArrayIterator
+{
+    public function __construct(iterable $iterator)
+    {
+        parent::__construct(array_reverse(iterator_to_array($iterator)));
```

Comment:
The SortableIterator is using `iterator_to_array($this->iterator, true)`. So should we do the same here ?